qa/common/infer_util.py

```diff
@@ -65,7 +65,10 @@ def _prepend_string_size(input_values):
             # don't convert it to str as Python will encode the
             # bytes which may distort the meaning
             if obj.dtype.type == np.bytes_:
-                s = bytes(obj)
+                if type(obj.item()) == bytes:
+                    s = obj.item()
+                else:
+                    s = bytes(obj)
```

Comment:
Can we abstract all the processing code into a common functionality?

Reply:
Yes, I will do that

Reply:
For now I have kept them in isolated functions. Do we want to have a separate string utility file that does all this?

Reply:
After another look, this function and the `serialize_string_tensor()` function that calls it don't really belong to both the (CUDA) shared memory Python library. You should just remove them, and make the serialize function be an utility function provided by the `tensorrtserver.api` package, as all "client" needs this functionality will have to import `tensorrtserver.api` anyway

Reply:
Done